lib/siwapp_web/live/taxes_component.ex

```diff
@@ -113,4 +115,17 @@ defmodule SiwappWeb.TaxesComponent do
       Map.get(item.data, :taxes)
     end
   end
+
+  @spec unify_items_structure(map | list) :: map
+  defp unify_items_structure(%{"items" => %{}} = structure), do: structure
+
+  defp unify_items_structure(%{"items" => items} = structure) do
+    new_items =
+      items
+      |> Enum.with_index(0)
+      |> Enum.map(fn {value, index} -> {"#{index}", value} end)
+      |> Map.new()
```

Comment:
```suggestion
      |> Map.new(fn {value, index} -> {"#{index}", value} end)
```

Reply:
Why don't you use numeric indices? just asking…

Reply:
Because the idea is to normalize the response that sometimes happens with the normal one, and that is with indices being binaries. I've already suggested a whole refactor of this part and will be addressed in the future.